### DISCUSSION TITLE
Clipboard security issue

### DISCUSSION THREAD

**Discussion by sasaandonov** [rustdesk/rustdesk#9010] [Q&A] [answered]
### Bug Description

Before connection, if i copy some text and than connect to another computer, than user on another computer can paste my copied text. This is bug, you should count only clipboard after connection, not before connection.

### How to Reproduce

Just connect to another computer and paste text.

### Expected Behavior

Clipboard sholud be counted after connection, not before.

### Operating system(s) on local side and remote side

Ubuntu 24.04 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.7

### Screenshots

-

### Additional Context

I think this error is not operating system dependent and happens everywhere.

**Comment by rustdesk** [comment, marked as answer]
https://www.reddit.com/r/rustdesk/comments/1fkivem/clipboard_synchronization_before_connection/ 

https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/#sync-init-clipboard

https://github.com/rustdesk/rustdesk/commit/2e314bf032938fb57a82a8bed3285b490da5b9c9

Option modification is not ready on GUI yet.

**Comment by Neustradamus** [comment]
Any progress on it?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/14326#discussioncomment-15784200
